packages/pg/package.json

```diff
@@ -11,43 +11,55 @@
     "postgresql",
     "rdbms"
   ],
-  "homepage": "https://github.com/brianc/node-postgres",
+  "homepage": "git://github.com/daniel-liu-bitio/node-postgres.git",
   "repository": {
     "type": "git",
-    "url": "git://github.com/brianc/node-postgres.git",
+    "url": "git://github.com/daniel-liu-bitio/node-postgres.git",
     "directory": "packages/pg"
   },
-  "author": "Brian Carlson <brian.m.carlson@gmail.com>",
-  "main": "./lib",
-  "dependencies": {
-    "buffer-writer": "2.0.0",
-    "packet-reader": "1.0.0",
-    "pg-connection-string": "^2.5.0",
-    "pg-pool": "^3.5.1",
-    "pg-protocol": "^1.5.0",
-    "pg-types": "^2.1.0",
-    "pgpass": "1.x"
+  "author": "Daniel Liu",
+  "main": "./dist/webpack-bundle.js",
+  "browser": {
+    "fs": false
   },
   "devDependencies": {
-    "async": "0.9.0",
+    "@i2labs/dns": "^1.0.1",
+    "async": "^0.9.0",
     "bluebird": "3.5.2",
+    "buffer": "^6.0.3",
     "co": "4.6.0",
-    "pg-copy-streams": "0.3.0"
-  },
-  "peerDependencies": {
-    "pg-native": ">=2.0.0"
+    "crypto-browserify": "^3.12.0",
+    "node-polyfill-webpack-plugin": "^1.1.4",
+    "path-browserify": "^1.0.1",
+    "pg-copy-streams": "0.3.0",
+    "pg-protocol": "^1.5.0",
+    "pgpass": "^1.0.5",
+    "setimmediate": "^1.0.5",
+    "stream-browserify": "^3.0.0",
+    "tls-browserify": "^0.2.2",
+    "webpack": "^5.68.0",
+    "webpack-bundle-analyzer": "^4.5.0",
+    "webpack-cli": "^4.9.2",
+    "websocket-stream": "^5.5.2"
```

Comment:
I suspect the larger sticking point for a successful PR to node-postgres is the increased set of dependencies this would create? @terry-bit-io it would be awesome for this to be part of node-postgres if there's a way we can propose to do that easily (so we don't start to diverge from future contributions, too).